src/content/docs/devtools/Docker/docker-case-study

```diff
@@ -0,0 +1,79 @@
+author: Christian Grøtan <Chrissebah>
+tags: Docker, case study, DevTools
+---
+
+## Introduction
+
+Docker, developed by Docker Inc., is a suite of platform-as-a-service offerings that utilize OS-level virtualization to distribute software in packages, known as containers. Docker provides both free and premium tiers of service. The core component responsible for hosting these containers is referred to as Docker Engine, with its initial release dating back to 2013.
+
+## Brief History
+
+- 2010: Docker was created by a man called Solomon Hykes, and was initially intented to be an internal project at a complany called dotCloud.
+- 2013: In March of 2013 the first version of Docker, the Docker Engine, was released. It was released as an open-source project.
+- 2013: Later this same year in October, dotCloud officially changed its name to Docker, Inc. and shifted its main focus on the Docker Engine.
+- 2014: The introduction of Docker Hub came in June of 2014. Docker Hub is a cloud-based service made to allow users to share and manage containerised applications.
+- 2016: This year Microsoft announced that Docker was now available for use natively on Windows 10.
+- 2021: Docker Desktop for Windows and MacOS users was no longer free for enterprise users, introducing a personal plan for companies and adding a premium tier for the software. The Linux desktop app remained unaffected.
+- 2023: Docker acquired AtomicJar, an application for development testing.
+
+## Main Features
+
+The primary purpose of Docker is to solve challenges associated with deploying software in diverse computing environments. Its main features include:
+
+- **Dependency Management:**  Docker allows developers to encapsulate their applications along with all dependencies, libraries, and runtime environments into lightweight containers, ensuring consistency across different environments.
+- **Developer Productivity:** Docker streamlines the development workflow by providing tools for building, testing, and deploying applications within containers, enhancing collaboration among team members.
+- **Isolation:** Docker containers provide process isolation, enhancing security and stability by allowing multiple applications to run independently on the same host.
+- **Portability:** Docker containers are platform-independent, enabling developers to build applications once and deploy them anywhere, reducing compatibility issues.
+- **Resource Efficiency:** Docker containers share the host system's kernel, making them lightweight and efficient compared to traditional virtual machines, leading to faster deployment times.
+
+## Market Comparison
+
+For my comparison, I will compare Docker to possibly the biggest competitor, Kubernetes. Kubernetes is an open-source container orchestration system for automating software deployment, scaling, and management. Originally designed by Google, the project is now maintained by the Cloud Native Computing Foundation [Wikipedia]
```

Comment:
This sounds subjective, and the word I is the culprit. You need to make it objective. Make it sound third-person.